integration-tests/utils/integration-helpers.js

```diff
@@ -184,6 +207,26 @@ const deleteRuntimes = async ({ page, billingProject, workspaceName }) => {
   console.info(`deleted runtimes: ${deletedRuntimes}`);
 };
 
+const deleteRuntimesV2 = async ({ page, billingProject, workspaceName }) => {
+  const deletedRuntimes = await page.evaluate(
+    async (workspaceName) => {
+      const {
+        workspace: { workspaceId },
+      } = await window.Ajax().Workspaces.workspaceV2(billingProject, workspaceName).details(['workspace.workspaceId']);
+      const runtimes = await window.Ajax().Runtimes.listV2WithWorkspace(workspaceId, { role: 'creator' });
+      return Promise.all(
+        _.map(async (runtime) => {
+          await window.Ajax().Runtimes.runtimeV2(runtime.workspaceId, runtime.runtimeName).delete(true); // true = also delete persistent disk.
+          return runtime.runtimeName;
+        }, _.remove({ status: 'Deleting' }, runtimes))
+      );
+    },
+    billingProject,
```

Comment:
The function passed to `page.evaluate` only takes one argument. It looks like this is passing `billingProject` as `workspaceName`.